cmd/tracee-ebpf/main.go

```diff
@@ -299,6 +311,11 @@ func main() {
 				Value:   cli.NewStringSlice("none"),
 				Usage:   "Control event caching queues. run '--cache help' for more info.",
 			},
+			&cli.StringSliceFlag{
+				Name:  "containers",
```

Comment:
Yeah i've had the same feeling, but container-runtime is also a bit of a mouthful. I thought of `sockets` as an alternative but it's also a bit unexpressive of the actual intention.
Maybe a third opinion can decide, because I don't really like any of these options.

Reply:
I think runtime[s] might be a better fit than containers. One reason for this is that containers can be confused with the containers filter. WDYT?

Reply:
No problem with that, just to make sure you mean `--runtimes` right?

Reply:
yes

Reply:
This is still something we might want to change.
Maybe we should call it "crs" (container runtime socket)? It is both short and more correct than just "container" or "runtime"